Cargo.toml

```diff
@@ -57,7 +57,7 @@ sqlx = {version = "0", features = [
 ]}
 
 helium-crypto = {version = "0.6.8", features=["sqlx-postgres", "multisig"]}
-helium-proto = {git = "https://github.com/helium/proto", branch = "master", features = ["services"]}
+helium-proto = {git = "https://github.com/helium/proto", branch = "map/modeled-coverage", features = ["services"]}
```

Comment:
just a note to update the `beacon` branch of proto at same time otherwise compile fails due to multiple version of proto in use